### DISCUSSION TITLE
"메타버스"가 무엇일까요?

### DISCUSSION THREAD

**Discussion by x4cf** [naverz/zepeto-studio-kor#693] [4. General]
메타버스란 무엇일까요?
-

> 요즘 많이 들려오는 단어인데, 왜 핫한 걸까요?
>  왜 새로운 카테고리가 필요한 것이고, 이전의 서비스와는 정말 다른 걸까요?

위키백과에서는 "차원에서 실제 생활과 법적으로 인정한 활동인 직업, 금융, 학습 등이 연결된...", 네이버 국어사전에서는 "웹상에서 아바타를 이용하여 사회, 경제, 문화적 활동을 하는 따위처럼 가상 세계와 현실 세계의 경계가 허물어지는..."이라고 정의하고 있는데요...
  
개발자 또는 유저분들에게는 잘 와닿으시나요?  새로운 기술이라는데 기대되는 서비스가 있나요?

  저는 저런 설명을 들으면 그 어떤 것도 기대되지 않아요. 
그럼 메타버스 기술은 그저 허풍, 사기일까요?  

메타버스를 알아보기 위해서 메타버스가 아닌 것들을 다시 생각해 보려고 해요 :)

우리가 기존에 알고 있던 게임 서비스로 시작합니다.
우리는 A라는 게임을 즐기기 위해서 게임사 A의 패키지를 구매하고 게임을 설치했습니다.
 A라는 게임은 플레이어가 주인공 “제임스”가 되어서 보물을 찾아 떠나는 게임입니다. 
정말 흥미진진한 모험이 되겠군요!  

우리는 험난한 정글로 고대 유적을 찾아 떠났고, 유적지에서 찾은 보물을 빼앗기 위해 나타난 클라임 박사와의 추격전에서 헬리콥터를 멋지게 따돌렸어요!
 우리의 주인공 제임스는 대륙으로 돌아와 아내와 함께 포옹을 하는 것으로 훈훈한 마무리가 지어졌습니다 :)  

다음 주 주말에 우리는 기억합니다. “1주일 전의 모험은 정말 멋졌어!” 
새로운 모험을 떠나기 위해서 우리는 새로운 게임을 찾다가 B라는 게임을 찾았습니다.

우리는 B라는 패키지를 구매하고 게임을 설치했습니다. 
B라는 게임은 플레이어가 주인공 “클라크”가 되어서 위험한 할렘가에서 갱단을 들어가는 것으로 시작합니다.
 이번에도 흥미진진한 모험이 될 것 같습니다! 
1주일 전의 저는 제임스였고 지금은 클라크이지만 문제는 없습니다 :)  

우리는 할렘가에서 태어난 클라크의 불우한 과거사를 경험하고 “가진 자가 정의다”라고 생각하며 정의롭기 위해, 정비소에서 만난 파트너와의 사랑으로 낳은 아이를 행복하게 살 수 있게 하기 위해서 극악무도한 행동도 마다하지 않습니다.
 하지만 클라크의 악행은 더 큰 악으로 다가왔고 결국 사랑하던 이들을 잃는 결과로 게임은 끝납니다. 
행복한 마무리는 아니었지만 그래도 인상적인 경험이었습니다.  

여기까지는 우리가 알고 있던 기존 게임 서비스의 이야기입니다. 
우리는 게임사에서 정해주는 다양한 인물이 되어 현실에서는 쉽게 경험하기 힘든 삶을 살아갑니다.

이번에는 메타버스의 시초라고 불리는 마인크래프트를 알아보겠습니다.  

우리는 마인크래프트를 구매하고 설치했습니다. 
제 캐릭터는 곰돌이 옷을 입은 플레이어 ABBA92가 되었습니다.  

오늘 우리는 눈을 뜨니 처음 보는 천장이 보입니다. 
나무로 된 저택에서 눈을 뜬 저는 머리맡에 있는 책을 펼쳐봅니다.
 이 세상에서 살아가야 할 규칙이 적혀있었고 우리는 침대에서 일어나 복도로 나갑니다.
 타이틀 게임처럼 멋진 그래픽은 아니었지만, 뜬금없이 괴물이 뛰쳐나오고 땅이 갈라지며 용암이 흘러들어옵니다. 
여전히 곰돌이 옷을 입고 ABBA92라는 캐릭터로 위험을 피하기 위해 움직입니다.

오늘은 또 새로운 곳입니다. 
사방이 돌로 된 지하에 갇혀 점점 물이 차오릅니다. 
여전히 곰돌이 옷을 입고 ABBA92라는 캐릭터로 말이죠.  
저는 마인크래프트라는 플랫폼에서 어디를 가든지 제 아바타로 모험을 떠납니다.
 콘텐츠에 따라, 크리에이터가 정해준 이야기의 등장인물이 되지만 그래도 제 캐릭터입니다.

  저는 메타버스가 “내 아바타가 주인공이 되어 여러 콘텐츠를 즐길 수 있는 플랫폼 서비스”라고 생각해요.
이전의 모든 콘텐츠와는 다르게 이제는 내가 주인공이 되는 것이죠 :D

요즘 인터넷 방송에서는 버추얼 아바타를 착용한 스트리머가 캐릭터를 “자신”이라고 말하고 캐릭터가 아파하면 소리를 지르며 깜짝 놀랍니다.
현 새대의 친구들은 이제 소셜 아바타와 자신을 동일시하며 캐릭터 또한 하나의 모습으로 생각합니다.
 개인과 자신, 자아에 대한 표현방식의 확장되어가는 시대의 흐름에, 메타버스가 얼마나 성장할지도 기대됩니다

여러분들은 어떻게 생각하시나요?
메타버스란 무엇일까요?

**Comment by mahoworld** [comment]
요즘 메타버스에 대해서는 저도 같은 생각입니다.

그러나 진정한 메타버스가 구현되는 순간은 오히려 미래에 대한 열망이나 진취적 태도에는 큰 관심이 없을 것 같아요.
아마도 미래 시기에 메타버스로 구현된 세계의 시점은 오히려 과거이지 않을까...싶네요.
인간은 그때쯤이면 과거의 추억들을 그리워할 것 같아요.
저도 미래에는 그럴 것 같구요....

대략 예상해서 기술이 발전해 2050년?2100년? 이후쯤 상용화가 된다면... 
그때는 극심한 기후변화와 환경파괴 때문에 오히려 지금 이 순간의 시대를 그리워하겠죠..? 
아마 탄소세같은 환경부담금 때문에 고기도 마음대로 못 먹어서 콩고기만 먹고 살아야 되고...에너지도 원하는 만큼 쓸 수 없을 것이고.. 
지금보다 훨씬 더 제약이 많은 게 미래 사회일텐데... 그런 대중들의 불만을 해소해주기 위해서 메타버스는 
인간이 가장 위대했던 순간으로 거슬러 올라가서 사람들의 불만족을 채워주는 형태로 존재하지 않을까 싶습니다.

지금은 인간이 우주로 나가 새로운 터전을 만들겠다는 진취적인 생각들을 많이 하지만
나중 미래 세대는 햇빛이 내리쬐는 숲 속에서 숨을 쉰다는 것은 어떤 것일까. 바다의 날생선 맛은 어떤 맛일까. 
이런 우리에겐 사소하지만 그들에겐 중요한 체험을 메타버스에서 오히려 더 원하지 않을까 싶어요.

저는 완전히 새로운 무언가를 만들어내는 출중한 실력은 없지만...
메타버스가 지금 세대와 다음 세대를 잇는 디지털적인 교두보가 되기를 바라면서
열심히 배우는 중입니다.

과연 메타버스는 어떤 방향으로 흘러가게 될 것인지!
저도 참 궁금하네요. 앞으로 큰 문제가 없다면 살 날이 많이 남은 사람으로서....

어쨌든 모두에게 즐거움을 줄 수 있는 방향으로 발전했으면 좋겠어요.

**Comment by ted002** [comment]
메타버스...결국에는 아바타와 플레이어간의 동화라고 저는 생각합니다.

얼마나 실감나는 그래픽이건 영상이건 결국엔 화면속 일, 현실의 자신에게 크게 영향이 오지는 않죠.

물론, 정신적으로 해당 캐릭터에 공감할수도 있고, 아닐수도 있죠. 그야말로 누구나 느끼는 점은 다를테니까요.

플레이어가 자기 자신을 아바타에 얼마나 이입해서 느끼느냐가 메타버스의 핵심이라고 봅니다.

자기 자신과 닮은 아바타, 자기 자신보다 더 훌륭한 아바타, 자신과 다르게 생긴 아바타.

하지만 누구나 자신만의 아바타에 공감하고, 자신만의 아바타에 이입해서, 다양한 세계를 느낄수 있죠.

최종적으로는 과거에 유행했던 장르소설 중 게임판타지 소설에 나왔었던것 처럼, 혹은 모 애니메이션(소드 아트 온라인)에서 나온것처럼, 혹은 영화 매트릭스에서 나온것 처럼, 완전히 오감마저도 지배하는 세상, 즉 가상현실이 진정한 도착지점이 아닐까 생각합니다.

물론, 여기까지 가기에는 많이 멀기는 합니다. 오감중 남은 세가지, 후각·미각·촉각중 제일 간단한 촉감조차도 구현에 난항을 겪고 있는 상황이고, 미각, 후각은 어불성설이죠.

그래서 저는 메타버스라는 개념이 나왔다고 합니다. 중간지점의 의미로요. 진정한 의미로의 가상현실이 나오기에는 멀고도 멀기에, 현재 여기까지 개발된 지점, 그리고 앞으로 혁신적인 개발포인트가 없는한, 이 모든 지점은 메타버스라고 지칭하는것이라고 생각합니다.
